pkg/apis/serving/metadata_validation_test.go

```diff
@@ -99,60 +98,63 @@ func TestValidateObjectMetadata(t *testing.T) {
 			Message: "name or generateName is required",
 			Paths:   []string{"name"},
 		},
-	}}
-
-	for _, c := range cases {
-		t.Run(c.name, func(t *testing.T) {
-
-			err := ValidateObjectMetadata(c.objectMeta)
-
-			if !reflect.DeepEqual(c.expectErr, err) {
-				t.Errorf("Expected: '%#v', Got: '%#v'", c.expectErr, err)
-			}
-		})
-	}
-}
-
-func TestAnnotationValidation(t *testing.T) {
```

Comment:
I wonder if it makes sense to still keep this test `TestAnnotationValidation`, and we just add one more test case to `TestValidateObjectMetadata` for an invalid annotation? This is just my personal preference. I'm thinking we can add one or more test case to `TestValidateObjectMetadata` for completeness, and test more comprehensively for invalid annotation in `TestAnnotationValidation`. What do you think?

Reply:
In this case, the one error case in `TestAnnotationValidation` was moved into `TestValidateObjectMetadata`. Since there was the non-error cases left, it made sense to move it into the same test and removed them. This also allowed us to test through the exported function.

Reply:
@andrew-su ahh got it! I wasn't reading carefully and made some assumptions